modules/imap/site.js

```diff
@@ -1,1102 +1,1320 @@
-'use strict';
+"use strict";
```

Comment:
There are quite many spacing changes here that are not related to the PR - please reset and only do the changes you want. E.g. changing from 4 to 2 spaces for the whole file is not desirable.

Reply:
This file is not yet resolved - see diff section of this PR.